src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Creates a new storage instance from the given pathname string.
+     * @param pathname A pathname string.
+     */
+    public Storage(String pathname) {
```

Comment:
the variable pathname should be pathName to follow the java coding standard (variable names should be written in camelCase)